angular/src/app/containers/form/form.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Inject,
+  OnInit
+} from '@angular/core';
+import {
+  FormBuilder,
+  FormGroup,
+} from '@angular/forms';
+import { Store } from '@ngrx/store';
+import * as reducers from '../../reducers';
+import { SetSelectionAction } from '../../actions/map';
+
+@Component({
+  selector: 'dp-form',
```

Comment:
Persoonlijk vind ik `dp-form` niet heel descriptive, terwijl de functionaliteit vrij specifiek is. Wellicht kun je dit hernoemen naar iets als `dp-timepicker`?